content/news/041/index.md

```diff
@@ -69,6 +69,41 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### Digital Extinction
+
+![laser trail in Digital Extinction](digital-extinction.jpeg)
+_Laser trail in Digital Extinction_
+
+[Digital Extinction] ([GitHub][de-github], [Discord][de-discord],
+[Reddit][de-reddit]) by [@Indy2222] is a 3D real-time strategy game made with
+[Bevy].
+
+This month's update is somewhat smaller but there has been some important
+progress in multiplayer.
+
+Here is the summary of the changes since the last update. It consists of commit
+range `28d1fce..3dbac4e` (2022-12-02–2022-12-29). There were 26 non-merge
```

Comment:
The section LGTM overall but the "It consists of commit ... commits in total" part seems unnecessary? or do you have some reason to be specified so precisely?

Reply:
Ok, removed.